llvm-general/src/LLVM/General/Internal/FFI/InstructionC.cpp

```diff
@@ -266,6 +266,15 @@ void LLVM_General_GetIndirectBrDests(
 		*dests = wrap(ib->getDestination(i));
 }
 
+inline Metadata **unwrap(LLVMMetadataRef *Vals) {
```

Comment:
I've used haskellish rather than LLVMish conventions for my C functions - e.g. lower case variable names. Please try to maintain consistency. If you want to argue for using LLVM conventions instead, that's fine but would be better as a project-wide PR doing just that.